TeachingRecordSystem/src/TeachingRecordSystem.AuthorizeAccess/Pages/RequestTrn/EmailInUse.cshtml.cs

```diff
@@ -0,0 +1,15 @@
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using TeachingRecordSystem.UiCommon.FormFlow;
+
+namespace TeachingRecordSystem.AuthorizeAccess.Pages.RequestTrn;
+
+[Journey(RequestTrnJourneyState.JourneyName), RequireJourneyInstance]
+public class EmailInUseModel : PageModel
```

Comment:
Need tests for this page. To be consistent with other pages it should also check that `Email` has been populated and redirect back if not.